app/main.py

```diff
@@ -1,6 +1,26 @@
+import requests
+from time import sleep
+import os
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    API_KEY = os.getenv("API_KEY")
+    URL = "http://api.weatherapi.com/v1/current.json?"
+    CITY = "Paris"
+    response = requests.get(URL + f"key={API_KEY}" + f"&q={CITY}")
```

Comment:
FYI: you can use params instead of hardcoding query params

![image](https://user-images.githubusercontent.com/44442087/221359456-5ff56377-1662-469a-af5e-1b11f7c08767.png)